bootcamp/tutorials/integration/llama_agents_metadata.ipynb

```diff
@@ -0,0 +1,767 @@
+{
```

Comment:
<p>the visual is very intuitive. good point.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/milvus-io/bootcamp/pull/1401/discussion/#comment-1705463975'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>